static/app/views/performance/landing/widgets/widgets/vitalWidgetMetrics.tsx

```diff
@@ -50,6 +50,7 @@ export function VitalWidgetMetrics(props: PerformanceWidgetProps) {
   const [selectedListIndex, setSelectListIndex] = useState(0);
   const field = props.fields[0];
   const vital = settingToVital[chartSetting];
+  const orgSlug = organization.slug;
```

Comment:
Probably fine to just pass organization.slug and keep the code more straightforward. Probably something to talk about at a TSC 🤔

Reply:
when `organization.slug` is often repeated in the same file, I usually like to create a reusable variable .. but this is just a preference thing... we can for sure discuss this in the TSC meeting 😉